low-level/framework/messaging/ingress_processor.py

```diff
@@ -257,7 +257,7 @@ def _read_config(self):
         self._consumer_id = Conf.get(SSPL_CONF,
                                      f"{self.PROCESSOR}>{self.CONSUMER_ID}",
                                      'sspl_actuator')
-        self._consumer_group = socket.getfqdn()
+        self._consumer_group = socket.gethostname()
```

Comment:
why this needed ?

Reply:
In order to broadcast request to each node we need to assign a unique consumer group to each consumer node. We can use any value which is unique for a given node across cluster. Like machine-id, hostname etc.